src/controller.ts

```diff
@@ -6,6 +6,8 @@ import { AustinProfileTaskProvider } from './providers/task';
 
 
 export class AustinController {
+    private output = vscode.window.createOutputChannel("Austin");
```

Comment:
It looks like this is the culprit. Probably something to do with creating a duplicate channel

Reply:
I'll raise a PR with a singleton logging instance

Reply:
Just saw https://github.com/P403n1x87/austin-vscode/commit/19141d33b010ec99048c3b2b84dbc082c8022320 so its already done